### PR TITLE
Complete Config

### PR DESCRIPTION
**Pull request by fosskers**

### CODE REVIEW

File: complete-config.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      - key: component
+        value: local-handler
+        level: info
+    default: error
+
+# --- Miscellaneous Settings --- #
+
+# Location of your database files. Leaving this as `null` tells Chainweb to use
+# its default locations.
+databaseDirectory: null
```
Comment (mightybyte) [inline complete-config.yaml]: Should we take this out or suggest that people not use this until the DB location bug is fixed?

Comment (fosskers) [reply complete-config.yaml]: The fix only applies to the default location, I think. @larskuhtz ?

Comment (mightybyte) [reply complete-config.yaml]: The default location puts the DBs in the right place.  I believe the bug happens when people override it.

Comment (fosskers) [reply complete-config.yaml]: I'll hide it for now.

**Pull request merged**